openeo/local/processing.py

```diff
@@ -59,4 +68,74 @@ def load_local_collection(*args, **kwargs):
 PROCESS_REGISTRY["load_collection"] = Process(
     spec=openeo_processes_dask.specs.load_collection, 
     implementation=load_local_collection,
+)
+
+def resample_cube_spatial_rioxarray(*args, **kwargs):
+    pretty_args = {k: type(v) for k, v in kwargs.items()}
+    _log.debug(f"Running process resample_cube_spatial")
+    _log.debug(f"kwargs: {pretty_args}")
+    _log.debug("-" * 80)
+
+    methods_list = {
+        "near": 0,
+        "bilinear": 1,
+        "cubic": 2,
+        "cubicspline": 3,
+        "lanczos": 4,
+        "average": 5,
+        "mode": 6,
+        "gauss": 7,
+        "max": 8,
+        "min": 9,
+        "med": 10,
+        "q1": 11,
+        "q3": 12,
+        "sum": 13,
+        "rms": 14
+    }
+    data = kwargs['data']
+    target = kwargs['target']
+    method = kwargs['method']
+    # The target doesn't need to have the same dimensions as the data
+    input_data_dims = (
+        data.openeo.band_dims + data.openeo.temporal_dims + data.openeo.spatial_dims + data.openeo.other_dims
+    )
+
+    if method not in methods_list.keys():
```

Comment:
FYI: with dicts you can just do 
```suggestion
    if method not in methods_list:
```